pkg/provider/aws/resources/ecr.go

```diff
@@ -50,6 +50,16 @@ func (repo *EcrRepository) Create(dag *core.ResourceGraph, params RepoCreatePara
 	return nil
 }
 
+type EcrRepositoryConfigureParams struct {
+	ForceDelete bool
+}
+
+// Configure sets the intristic characteristics of a vpc based on parameters passed in
+func (repo *EcrRepository) Configure(params EcrRepositoryConfigureParams) error {
+	repo.ForceDelete = true
```

Comment:
i will bring this up in standup, but i think we may want to start tackling design on the constraints. For most i have it set the params if its a non0 value but bool is a tricky one. We dont want to set it to false all the time because then we cant destroy stacks